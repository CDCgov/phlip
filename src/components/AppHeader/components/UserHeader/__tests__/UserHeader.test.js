import React from 'react'
import { shallow } from 'enzyme'
import { UserHeader } from '../index'

const props = {
  user: { firstName: 'Test', lastName: 'User', role: 'Coordinator' },
  open: false,
  handleLogoutUser: jest.fn(),
  handleToggleMenu: jest.fn(),
  handleCloseMenu: jest.fn(),
  handleOpenHelpPdf: jest.fn(),
  handleOpenAdminPage: jest.fn()
}

describe('UserHeader', () => {
  test('should render correctly', () => {
    expect(shallow(<UserHeader {...props} />)).toMatchSnapshot()
  })
})