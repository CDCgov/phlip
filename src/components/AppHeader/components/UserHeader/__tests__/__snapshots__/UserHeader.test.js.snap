// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserHeader should render correctly 1`] = `
<FlexGrid
  align="center"
  circular={false}
  container={true}
  flex={false}
  justify="stretch"
  padding={0}
  raised={false}
  style={Object {}}
  type="row"
>
  <FlexGrid
    align="stretch"
    circular={false}
    container={false}
    flex={false}
    justify="stretch"
    padding={8}
    raised={false}
    style={Object {}}
    type="column"
  >
    <Greeting
      firstName="Test"
      lastName="User"
      role="Coordinator"
    />
  </FlexGrid>
  <AvatarMenu
    initials="TU"
    onLogoutUser={[MockFunction]}
    onOpenAdminPage={[MockFunction]}
    onOpenHelpPdf={[MockFunction]}
    onToggleMenu={[MockFunction]}
    open={false}
    role="Coordinator"
    userName="Test User"
  />
</FlexGrid>
`;
